setup.cfg

```diff
@@ -31,6 +31,7 @@ install_requires =
     pytest-reraise
     pynetconsole>=2.0.2
     pint>=0.11.0
+    gitpython>=3.1
```

Comment:
According to the repo:

> GitPython needs the git executable to be installed on the system and available in your PATH for most operations.

... we can also just call `subprocess.run(['git', 'describe', '--dirty=-dirty'])` for something simple like this.

Reply:
This should be done now